plugins/dell-dock/README.md

```diff
@@ -48,6 +48,12 @@ These devices use several different generation schemes, e.g.
  * MST Hub: `MST-panamera-vmm5331-259`
  * Thunderbolt Controller: `TBT-00d4b070`
 
+Update Behavior
+---------------
+
+For devices implementing "passive flow" the update is deployed at runtime and
+then actually written the next time the dock is attached.
```

Comment:
Come to think of it, NOT using passive was only pre-production devices.  I wonder if it's worth just ripping out the code for non-passive update now.

Reply:
I think ripping out code for immediate update (on the assumption they're not what was ever sold in shops) is the best plan. Do you mind doing that and updating the README please? You can probably describe the delayed write thing better than me.

Reply:
The main thing needed for passive flow was a new enough EC and a new enough hub.  The earliest hub in public release was 01.47 (https://fwupd.org/lvfs/devices/com.dell.wd198ceeeffd.firmware) and minimum EC was 00.00.00.23 (torn out here https://github.com/fwupd/fwupd/commit/ead0deaa255f08db0d777deb97f650d10d545eb5#diff-2d320a289c64c6066499373624e1d9b944d4e13b0eccd3068dad84fda2afd804).

I'll finish tearing out the rest then.